docs/index.md

```diff
@@ -22,7 +22,13 @@ If you set up BTCPay using any dockerized install method, you can enable BTC Tra
 ```bash
 sudo su -
 cd btcpayserver-docker
-export BTCPAYGEN_ADDITIONAL_FRAGMENTS="opt-add-btctransmuter"
+
+# retrieve your currently used environment variables
+echo $BTCPAYGEN_ADDITIONAL_FRAGMENTS
+
+# add btcTransmuter to your environment variables list that is shown (example below: pruning, btcTransmuter)
+export BTCPAYGEN_ADDITIONAL_FRAGMENTS="opt-save-storage-s;opt-add-btctransmuter"
```

Comment:
` export BTCPAYGEN_ADDITIONAL_FRAGMENTS="$BTCPAYGEN_ADDITIONAL_FRAGMENTS;opt-add-btctransmuter" `
is easier